flat-manager/index.js

```diff
@@ -12,13 +12,30 @@ const run = (repository, flatManagerUrl, token) => {
   ])
     .then(async () => {
       let buildId = ''
-      const exitCode = await exec.exec('flat-manager-client', [
+      let args = [
         '--token',
-        token,
+        token  
+      ]
+
+      if (endOfLive) {
```

Comment:
Probably should be endOfLife. You also forgot to pass these parameters to the run function above

Reply:
I'm very sorry that I overlook so much 🙈